src/Text/Pandoc/Writers/Markdown.hs

```diff
@@ -77,6 +81,25 @@ writeCommonMark opts document =
                    enableExtension Ext_intraword_underscores $
                      writerExtensions opts }
 
+-- | Convert Pandoc to Markua.
+writeMarkua :: PandocMonad m => WriterOptions -> Pandoc -> m Text
+writeMarkua opts document =
+  evalMD (pandocToMarkdown opts' document) def{ envVariant = Markua } def
+ where
+  opts' = opts{   writerExtensions =
+                  enableExtension Ext_hard_line_breaks $
+                  enableExtension Ext_pipe_tables $
+                  enableExtension Ext_fancy_lists $
+                  enableExtension Ext_startnum $
+                  enableExtension Ext_strikeout $
+                  enableExtension Ext_subscript $
+                  enableExtension Ext_superscript $
+                  enableExtension Ext_definition_lists $
+                  enableExtension Ext_smart $
+                  enableExtension Ext_footnotes $
+                    writerExtensions opts }
```

Comment:
Question: does it make sense to have *any* variation in extensions for Markua, given that the whole format is specified?  E.g. would it ever make sense to add the `grid_tables` extension?  If not, then maybe best simply to hard code a list of extensions here.

Reply:
We agree, that it doesn't make much sense to change certain options. However we enabled some, that will create the correct version also used by markdown (ie. pipetables) in order not to write duplicate code.

Reply:
I understand; I wasn't suggesting that you not enable the extensions here.
The question is whether you should be adding them to `writerExtensions opts`, which implies that users can set extensions not on this list.  (In other words, could `writerExtensions opts` be replaced by `mempty`?  If there are no writer options it makes sense for the *user* to enable, then I think it could.)

Reply:
we changed it